tools/cmake.py

```diff
@@ -1,13 +1,20 @@
-import os, sys
+import os
+import shlex
+import sys
 
-import SCons.Util
-import SCons.Builder
 import SCons.Action
+import SCons.Builder
+import SCons.Util
+
+
+# This must be kept in sync with the value in https://github.com/godotengine/godot/blob/master/platform/android/detect.py#L58.
+def get_ndk_version():
+    return "23.2.8568313"
```

Comment:
The link is no longer accurate. This one is better:
https://github.com/godotengine/godot/blob/master/platform/android/java/app/config.gradle#L16

And we're now on NDK r28 upstream, though I don't know how in sync the Git plugin needs to be...

Actually, we don't build this plugin for Android, so I guess this is just copied from another cross-platform repo?

Reply:
Yeah, this comes from the webrtc-native repository.

I've been wondering how we can maintain GDExtension compatibility.... what happens if we build with a different NDK version? Is targeting the lowest NDK guaranteed to work? AFAIK using a newer version results in the library failing to load.